### ISSUE TITLE
unrecognized arguments: --dataset_kwargs {

### ISSUE THREAD

**Issue by lab9001** [AllenCellModeling/pytorch_fnet#118]
Hi there,

we have an error: unrecognized arguments. Do you guys have any ideas why this occurs?

We are also confused, which train_model.py script is the newest one? 

+ python train_model.py --n_iter 50000 --path_dataset_csv data/csvs/dna/train.csv --dataset_kwargs '{"transform_signal": ["fnet.transforms.Normalize()", "fnet.transforms.Resizer((1, 0.37241, 0.37241))"], "transform_target": ["fnet.transforms.Normalize()", "fnet.transforms.Resizer((1, 0.37241, 0.37241))"] }' --bpds_kwargs '{"buffer_size": 1 }' --path_run_dir saved_models/dna --gpu_ids 0
usage: train_model.py [-h] [--batch_size BATCH_SIZE]
                      [--bpds_kwargs BPDS_KWARGS] [--buffer_size BUFFER_SIZE]
                      [--buffer_switch_frequency BUFFER_SWITCH_FREQUENCY]
                      [--class_dataset CLASS_DATASET]
                      [--gpu_ids GPU_IDS [GPU_IDS ...]]
                      [--interval_save INTERVAL_SAVE]
                      [--iter_checkpoint ITER_CHECKPOINT [ITER_CHECKPOINT ...]]
                      [--lr LR] [--n_iter N_ITER] [--nn_kwargs NN_KWARGS]
                      [--nn_module NN_MODULE]
                      [--patch_size PATCH_SIZE [PATCH_SIZE ...]]
                      [--path_dataset_csv PATH_DATASET_CSV]
                      [--path_dataset_val_csv PATH_DATASET_VAL_CSV]
                      [--path_run_dir PATH_RUN_DIR] [--seed SEED]
                      [--shuffle_images]
                      [--transform_signal TRANSFORM_SIGNAL [TRANSFORM_SIGNAL ...]]
                      [--transform_target TRANSFORM_TARGET [TRANSFORM_TARGET ...]]
train_model.py: error: unrecognized arguments: --dataset_kwargs {"transform_signal": ["fnet.transforms.Normalize()", "fnet.transforms.Resizer((1, 0.37241, 0.37241))"], "transform_target": ["fnet.transforms.Normalize()", "fnet.transforms.Resizer((1, 0.37241, 0.37241))"] }

**Comment by counkomol** [comment]
You can see from the error message that the training script does not support the `--dataset_kwargs` argument. Can I ask where did you get this from?

**Comment by lab9001** [comment]
Thank you for your reply!
I just tried your code, and wanted to train a model, but in the current version is no train_model.py (not in the folder scripts) provided. So, after I startet the script train_model.sh it stopped, because there was no train_model.py. Then I used the train_model.py file from the "original paper" release branche1. Now, the train_model.sh started and the train_model.py was started by the shell script, but it stopped again due to the error message train_model.py: error: unrecognized arguments: --dataset_kwargs {"transform_signal": . 
Do you have any ideas who to solve this issue? Thank you very much in advance!

**Comment by counkomol** [comment]
You'll need to stick to one snapshot of the repository. For example, `release_1`. If you mix and match different files from different repository states, then there is no way for us to know what to expect.

We suggest you try a fresh copy of `release_1` again and let us know if you find anything.

**Comment by lab9001** [comment]
Thank you! I will stick to one repository. The issue is that there is no train_model.py file in the newest repository. Could your please provide it?

**Comment by counkomol** [comment]
The `train_model.py` file does not exist because that is intended. With that said, the newest version of the repository is in active development and is currently not meant for outside use.

**Comment by lab9001** [comment]
Ok, thank you!
